pyEpiabm/pyEpiabm/utility/covidsim_kernel.py

```diff
@@ -0,0 +1,31 @@
+#
+# Class of Covidsim-style gravity kernels
+#
+
+
+class Kernel:
+    """Class to create the gravity kernel used throughout
+    CovidSim in the spatial weighting.
+    """
+    @staticmethod
+    def weighting(distance: float, scale: float = 4, shape: float = 3.8):
+        """Returns the weighting given by the Covidsim distance
+        weighting. Default parameters use the Covidsim GB spatial data.
```

Comment:
Its not clear how the parameters are related to spatial data. If it's relevant, I would elaborate this docstring.